core/src/test/java/org/infinispan/test/JREBlocking.java

```diff
@@ -0,0 +1,26 @@
+package org.infinispan.test;
+
+import java.util.concurrent.BrokenBarrierException;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.CyclicBarrier;
+import java.util.concurrent.Exchanger;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+public class JREBlocking {
```

Comment:
I suggest a name with `Test` in it, maybe `TestBlocking`.
Also missing javadoc :)

Reply:
Sure.